content/ko/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -0,0 +1,399 @@
+---
+title: 노드에 파드 할당하기
+content_template: templates/concept
+weight: 30
+---
+
+
+{{% capture overview %}}
+
+{{< glossary_tooltip text="파드" term_id="pod" >}}를 특정한 {{< glossary_tooltip text="노드(emf)" term_id="node" >}}에서만 동작하도록 하거나,
+특정 노드들을 선호하도록 제한할 수 있다.
+이를 수행하는 방법에는 여러 가지가 있으며, 권장되는 접근 방식은 모두
+[레이블 셀렉터(label selector)](/ko/docs/concepts/overview/working-with-objects/labels/)를 사용하여 선택한다.
+보통 스케줄러가 자동으로 합리적인 배치를 수행하기에 이런 제약 조건은 필요하지 않지만
+(예: 노드들에 걸쳐 파드를 분배하거나, 여유 자원이 부족한 노드에 파드를 배치하는 등)
+간혹 파드가 착륙하는 노드에 대해 더 많은 제어를 원할 수 있는 상황이 있다.
+예를 들어 SSD가 장착된 머신에 파드가 연결되도록 하거나 또는 동일한 가용성 영역(availability zone)에서
+많은 것을 통신하는 두 개의 서로 다른 서비스의 파드를 같이 배치할 수 있다.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## 노드 셀렉터(nodeSelector)
+
+`nodeSelector` 는 가장 간단하고 권장하는 노드 선택 제약 조건의 형태이다.
+`nodeSelector` 는 PodSpec의 필드이다. 이는 키-값 쌍의 매핑으로 지정한다. 파드가 노드에서 동작할 수 있으려면,
+노드는 키-값의 쌍으로 표시되는 레이블을 각자 가지고 있어야 한다(이는 추가 레이블을 가지고 있을 수 있다).
+대부분의 일반적인 사용은 하나의 키-값 쌍이다.
+
+`nodeSelector` 를 어떻게 사용하는지 예시를 통해 알아보도록 하자.
+
+### 0 단계: 사전 준비
+
+이 예시는 쿠버네티스 파드에에 대한 기본적인 이해를 하고 있고 [쿠버네티스 클러스터가 설정](/ko/docs/setup/)되어 있다고 가정한다.
+
+### 1 단계: 노드에 레이블 붙이기
+
+`kubectl get nodes` 를 실행해서 클러스터 노드 이름을 가져온다. 이 중에 레이블을 추가하기 원하는 것 하나를 선택한 다음에 `kubectl label nodes <노드 이름> <레이블 키>=<레이블 값>` 을 실행해서 선택한 노드에 레이블을 추가한다. 예를 들어 노드의 이름이 'kubernetes-foo-node-1.c.a-robinson.internal' 이고, 원하는 레이블이 'disktype=ssd' 라면, `kubectl label nodes ubernetes-foo-node-1.c.a-robinson.internal disktype=ssd` 를 실행한다.
+
+`kubectl get nodes --show-labels` 를 다시 실행해서 노드가 현재 가진 레이블을 확인하여, 이 작업을 검증할 수 있다. 또한 `kubectl describe node "노드 이름"` 을 사용해서 노드에 주어진 레이블의 전체 목록을 확인할 수 있다.
+
+### 2 단계: 파드 설정에 nodeSelector 필드 추가하기
+
+실행하고자 하는 파드의 설정 파일을 가져오고, 이처럼 nodeSelector 섹션을 추가한다. 예를 들어 이것이 파드 설정이라면:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+  labels:
+    env: test
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+```
+
+이 다음에 nodeSelector 를 다음과 같이 추가한다:
+
+{{< codenew file="pods/pod-nginx.yaml" >}}
+
+그런 다음에 `kubectl apply -f https://k8s.io/examples/pods/pod-nginx.yaml` 을
+실행하면, 레이블이 붙여진 노드에 파드가 스케줄 된다.
+`kubectl get pods -o wide` 를 실행해서 파드가 할당된
+"NODE" 를 보면 작동하는지 검증할 수 있다.
+
+## 넘어가기 전에: 내장 노드 레이블들 {#built-in-node-labels}

```

Comment:
Possibly use a Korean `id` attribute?
```suggestion
## 넘어가기 전에: 내장 노드 레이블들
```

Reply:
@sftim 
The Korean language team decided on this part at the regular meeting on November 7, 2019.
"Identifiers overridden in English documents are handled in English." - [Link](https://goo.gl/1gZ45u)
The reason for this is that we talked about the same thing in document # 16889.